recipe-server/package.json

```diff
@@ -109,14 +111,14 @@
     "redux-devtools-log-monitor": "1.0.11",
     "redux-logger": "3.0.6",
     "redux-mock-store": "1.1.4",
-    "sass-loader": "4.0.1",
+    "sass-loader": "6.0.6",
     "style-loader": "0.13.1",
     "stylelint": "7.6.0",
     "stylelint-config-standard": "15.0.0",
     "stylelint-order": "0.2.2",
-    "webpack": "1.13.2",
+    "uglifyjs-webpack-plugin": "^0.4.6",
```

Comment:
I went with UglifyJS because I was aiming more for just condensing the vendor code than try to do any further transformations. I will see if we can use babili on non-babel code and swap it out accordingly.